block.go

```diff
@@ -1155,21 +1155,14 @@ func (p *parser) fencedCode(out *bytes.Buffer, data []byte, doRender bool) int {
 		beg = end
 	}
 	var caption bytes.Buffer
-	line := beg
 	j := beg
-	if bytes.HasPrefix(bytes.TrimSpace(data[j:]), []byte("Figure: ")) {
```

Comment:
while I welcome changes in this code path as well (just fix an annoying bug here myself)... Is this related to the current PR?